### DISCUSSION TITLE
Unable to build with GitHub pages

### DISCUSSION THREAD

**Discussion by eidoriantan** [community/community#23960] [Other Feature Feedback, Questions, & Ideas]
<p>I have a site deployed with GitHub pages. But when I updated some assets and docs, it’s now unable to build the page. I already tried running it locally with Jekyll and re-installed the Gemfile and it runs just fine.</p>
<p>Here’s the <a href="https://github.com/eidoriantan/mp3tag.js/tree/gh-pages" rel="noopener nofollow ugc">link to my repository</a>.</p>

**Comment by mathcat4** [comment]
<p>I think you have to name the repo <a href="http://your-username.github.io" rel="noopener nofollow ugc">your-username.github.io</a>. Then it will take 1-10 min and will hopefully work.</p>

**Comment by eidoriantan** [comment]
<p>No, it already worked the last time. As what you can see on the repository, there’s a <code>CNAME</code> file that I used for custom domain and it’s running at <a href="https://mp3tag.js.org" rel="noopener nofollow ugc">mp3tag.js.org</a>.</p>
<p>It’s just that when I updated an asset, it doesn’t build and update the website anymore.</p>
<p>Edit: It is also under the <code>gh-pages</code> branch and should enable GitHub pages to build the site.</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hey <a class="mention" href="/eidoriantan">@eidoriantan</a>, looking at the build logs I’m seeing this error for your site:</p>
<pre><code class="lang-auto">  Liquid Exception: Couldn't find octicon symbol for \\"cloud-download\\" in editor/index.html
             Fatal: RuntimeError
                    Couldn't find octicon symbol for \\"cloud-download\\"
</code></pre>
<p>This corresponds with this file: <a href="https://github.com/eidoriantan/mp3tag.js/blob/92c982d6ce321fa8637da49bc0d5f30a1749628c/editor/index.html#L109" class="inline-onebox">mp3tag.js/index.html at 92c982d6ce321fa8637da49bc0d5f30a1749628c · eidoriantan/mp3tag.js · GitHub</a></p>
<p>I’m wondering if the version of <code>jekyll-octicons</code> that Pages uses has been updated above v10, it looks like <a href="https://github.com/primer/octicons/releases/tag/v10.0.0">there was a breaking change</a> after that release that changed <code>cloud-download</code> and <code>cloud-upload</code> to just <code>download</code> and <code>upload</code> respectively.</p>

**Comment by eidoriantan** [comment]
<p>Thank you for your response</p>

**Comment by otter-computer** [comment]
<p>I can’t find any <em>official</em> documentation on this just yet so I’m not sure if that’ll actually fix it, but I’m hoping it’s somewhere to start? :laughing:</p>

**Comment by eidoriantan** [comment]
<p>Also, I’m not seeing the current version of <code>jekyll-octicons</code> on the <a href="https://pages.github.com/versions/" rel="noopener nofollow ugc">GitHub Pages dependencies</a> and so I wasn’t able to know what version the GitHub pages uses. It would really help if the version of <code>jekyll-octicons</code> is exposed to avoid this type of situations.</p>
<p>Anyway, thanks for the help!</p>

**Comment by Squirrel1489** [comment]
<h2>Github Page</h2>
<p>I was hoping to get mine done this month, how would you rate the process? <a class="mention" href="/eidoriantan">@eidoriantan</a></p>
